### PR TITLE
Feature ACA-2646 local ids

### PR DESCRIPTION
**Pull request by mmuller88**
Issue Number: ACA-2646

## Changes
Use ./docker-compose-keycloak.yml for IDS

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -32,16 +32,18 @@ jobs:
...
-    - name: Test Suite sharingContent&markFavorite
-      script: npm run build.e2e && SUITE="--suite sharingContent" npm run e2e:docker
-    - name: Test Suite viewContent&metadata&extensions
-      script: npm run build.e2e && SUITE="--suite viewer,infoDrawer,extensions" npm run e2e:docker
+    # - stage: e2e
+    #   name: Test Suite appNavigation&search
+    #   script: npm run build.e2e && SUITE="--suite authentication,listViews,navigation,application,pagination,search" npm run e2e:docker
+    - name: IDS Test Suite appNavigation&search
+      script: npm run build.e2e && ./start-sso.sh && SUITE="--suite authentication" npm run e2e
+    # - name: Test Suite actionsAvailable
```
Comment (DenysVuika) [inline .travis.yml]: remove commented out code if it is no longer required

File: docker-compose-keycloak.yml [hunk 2]
```diff
@@ -96,21 +155,39 @@ services:
       - 61616:61616 # OpenWire
       - 61613:61613 # STOMP
 
-  content-app:
+  content-app2:
```
Comment (adinapitul) [inline docker-compose-keycloak.yml]: don't think this second aca is needed.

File: .travis.yml (hunk 1 above)
Comment (mmuller88) [reply .travis.yml]: Was only for faster testing. Reverted that

File: .travis.yml [hunk 3]
```diff
@@ -35,6 +35,8 @@ jobs:
     - stage: e2e
       name: Test Suite appNavigation&search
       script: npm run build.e2e && SUITE="--suite authentication,listViews,navigation,application,pagination,search" npm run e2e:docker
+    # - name: IDS Test Suite appNavigation&search
```
Comment (DenysVuika) [inline .travis.yml]: we typically don't commit commented out code, remove it if it is no longer used

File: docker/proxy/nginx.conf [hunk 4]
```diff
@@ -23,6 +23,10 @@ server {
       proxy_pass http://alfresco:8080;
     }
 
+    location /api-explorer/ {
```
Comment (DenysVuika) [inline docker/proxy/nginx.conf]: why do we need api explorer for this compose file?

Comment (mmuller88) [reply docker/proxy/nginx.conf]: You could use it as an easy way to see how to communicate directly with ACS REST API or even use it for direct communication with using the try functionality from api-explorer. As well api-explorer included in the alfresco/alfresco-content-repository-community image. It cam in handy for AGS as we need to figure out the API call for creating the Share RM side.

**Comment by DenysVuika** [comment]
@mmuller88 the acceptance criteria in the Jira ticket says `Must use Identity Services`. This PR is not using ID but a vanilla keycloak (as was originally introduced). It that expected?

**Comment by mmuller88** [comment]
@DenysVuika great question. It seems we only have AIS in quay.io which would make qauy credentials necessary :-/

**Pull request merged**